go.mod

```diff
@@ -10,5 +10,7 @@ require (
 require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
+	github.com/russross/blackfriday/v2 v2.1.0 // indirect
+	golang.org/x/exp v0.0.0-20230522175609-2e198f4a06a1 // indirect
```

Comment:
This seems odd that `github.com/russross/blackfriday/v2` is re-introduced 🤔 It doesn't appear to be used.

I'm _more than a little bit_ nervous about including `golang.org/x/exp` given how volatile it is and having a pre-`v1` version that's more likely to differ with downstream `go.mod` files. Are the capabilities we're using something that can be copy-vendor'd into this codebase until it graduates out of `golang.org/x/exp`?

Reply:
Actually that is a very good point. Lets create a separate urfave/cli-validation library which has all this. That way the core repo is unaffected by all this. So the only addition in our repo is the Validator field and nothing else.

Reply:
ran go mod tidy and removed blackfriday